frontend/app/routes/protected/person-case/request-details.tsx

```diff
@@ -0,0 +1,151 @@
+        type: v.pipe(v.picklist(VALID_REQUESTS, t('protected:request-details.required-request'))),
+      }) satisfies v.GenericSchema<RequestDetailsSessionData>;
+
+      const input = {
+        scenario: formData.get('scenario') as string,
+        type: formData.get('request-type') as string,
+      } satisfies Partial<RequestDetailsSessionData>;
+
+      const parseResult = v.safeParse(schema, input, { lang });
+
+      if (!parseResult.success) {
+        return data({ errors: v.flatten<typeof schema>(parseResult.issues).nested }, { status: 400 });
+      }
+
+      context.session.inPersonSINCase ??= {};
+      context.session.inPersonSINCase.requestDetails = parseResult.output;
+
+      throw i18nRedirect('routes/protected/person-case/first-name.tsx', request);
+    }
+
+    default: {
+      throw new AppError(`Unrecognized action: ${action}`);
+    }
+  }
+}
+
+export default function CreateRequest({ loaderData, actionData, params }: Route.ComponentProps) {
+  const { t } = useTranslation(handle.i18nNamespace);
+  const fetcher = useFetcher<Info['actionData']>();
+  const isSubmitting = fetcher.state !== 'idle';
+  const errors = fetcher.data?.errors;
+  const scenarioOptions = VALID_SCENARIOS.map((value) => ({
+    value: value,
+    children: t(`protected:request-details.scenarios.${value}` as 'protected:request-details.scenarios.for-self'),
+    defaultChecked: value === loaderData.defaultFormValues.scenario,
+  }));
+  const requestOptions = ['select-option', ...VALID_REQUESTS].map((value) => ({
+    value: value === 'select-option' ? '' : value,
+    children: t(`protected:request-details.requests.${value}` as 'protected:request-details.requests.select-option'),
+  }));
+
+  return (
+    <>
+      <PageTitle subTitle={t('protected:in-person.title')}>{t('protected:request-details.page-title')}</PageTitle>
+      <ErrorSummary errors={errors} />
+      <fetcher.Form method="post" noValidate>
+        <div className="space-y-6">
+          <InputRadios
+            id="scenario"
+            legend={t('protected:request-details.select-scenario')}
+            name="scenario"
+            options={scenarioOptions}
+            required
+          />
+          <InputSelect
+            className="w-max rounded-sm"
+            id="request-type"
+            name="request-type"
+            label={t('protected:request-details.type-request')}
+            defaultValue={loaderData.defaultFormValues.type ?? ''}
+            options={requestOptions}
+          />
+          <div className="mt-8 flex flex-wrap items-center gap-3">
+            <Button name="action" value="back" id="back-button" disabled={isSubmitting}>
+              {t('protected:person-case.previous')}
+            </Button>
+            <Button name="action" value="next" variant="primary" id="continue-button" disabled={isSubmitting}>
+              {t('protected:person-case.next')}
+            </Button>
+          </div>
+        </div>
```

Comment:
Move action buttons `div` outside of this `div`.

```suggestion
        <div className="space-y-6">
          <InputRadios
            id="scenario"
            legend={t('protected:request-details.select-scenario')}
            name="scenario"
            options={scenarioOptions}
            required
          />
          <InputSelect
            className="w-max rounded-sm"
            id="request-type"
            name="request-type"
            label={t('protected:request-details.type-request')}
            defaultValue={loaderData.defaultFormValues.type ?? ''}
            options={requestOptions}
          />
        </div>
        <div className="mt-8 flex flex-wrap items-center gap-3">
          <Button name="action" value="back" id="back-button" disabled={isSubmitting}>
            {t('protected:person-case.previous')}
          </Button>
          <Button name="action" value="next" variant="primary" id="continue-button" disabled={isSubmitting}>
            {t('protected:person-case.next')}
          </Button>
        </div>
```